schemas/creativeWork.schema.tpl.json

```diff
@@ -0,0 +1,138 @@
+{
+  "properties": {
+    "about": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add links to the datasets, models or software contained within or described by the creative work.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DatasetVersion",
+        "https://openminds.ebrains.eu/core/ModelVersion",
+        "https://openminds.ebrains.eu/core/SoftwareVersion"
+      ]
+    },
```

Comment:
I agree with removing this property here and potentially adding creative work schemas as options to relatedPublications.

Theoretically though all creative works should be indirectly linked to the relatedPublications of the research products via their digital identifiers (DOI, ISBN, ISSN, etc). This would also avoid the issue if we should allow dependencies of openMINDS_core to extensions (which I think are probably not fully avoidable).